Verify and update translation to spanish of basic javascript

jorgearuv opened the pull request:
#6496

Please check this translations for challenges:
- [x] Storing Values with the Equal Operator.
- [x] Initializing Variables with the Equal Operator.
- [x] Understanding Uninitialized Variables.
- [x] Understanding Case Sensitivity in Variables.
- [x] Increment a Number with Javascript.
- [x] Decrement a Number with Javascript.
- [x] Finding a Remainder in Javascript.
- [x] Assignment with Plus Equals.
- [x] Assignment with Minus Equals.
- [x] Assignment with Times Equals.
- [x] Assignment with Divided by Equals.
- [x] Convert Celsius to Fahrenheit.

greenkeeperio-bot committed:
chore(package): update loopback-component-passport to version 2.0.0

http://greenkeeper.io/

greenkeeperio-bot committed:
chore(package): update dotenv to version 2.0.0

http://greenkeeper.io/

greenkeeperio-bot committed:
chore(package): update react-motion to version 0.4.2

http://greenkeeper.io/

greenkeeperio-bot committed:
chore(package): update cheerio to version 0.20.0

http://greenkeeper.io/

greenkeeperio-bot committed:
chore(package): update nodemailer to version 2.1.0

http://greenkeeper.io/

balthazar committed:
style(challenge00): use less imperative wording

vtamara committed:
Verified and updated translation to spanish of advanced-bonfires.
Closes #6494
Thanks for corrections to @justinian336

erictleung committed:
Clarify characters are each unique in permutation

alistermada committed:
Map and chat buttons toggle asides

MatheusGodinho committed:
Challenges of Json APIS and Ajax translated. Closes #6717

rfprod committed:
Russian Translation of React Projects. Closes #6808

Russian Translation of React Projects. Edited some typos. Closes #6808

Russian Translation of React Projects. Final edits.

vtamara committed:
Update translation to spanish of basic-bonfires
Closes #6495
Thanks for corrections to @justinian336 and @Narshe1412

jorgearuv committed:
Challenges of Basic Javascript translated.

jorgearuv committed:
Challenges of Basic Javascript translated.

SaintPeter committed:
Merge pull request #6798 from erictleung/fix/permutation-problem-explanation

Clarify characters are each unique in permutation

Narshe1412 commented:
All good here, translation seems accordingly

SaintPeter committed:
Merge pull request #6801 from alistermada/fix/map-button-on-active-map

Map and chat buttons close their respective asides when open

vtamara commented:
Reviewed. It is good translation. Good job @jorgeruvalcaba  and @Narshe1412

erictleung committed:
Clarify diff two arrays instructions

- Clarify you want items that are in one but not both arrays
- Add "symmetric difference" terminology for those familiar with it

SaintPeter committed:
Merge pull request #6841 from erictleung/fix/clarify-diff-two-arrays-instruction

Clarify diff two arrays instructions

erictleung committed:
Clarify symmetric difference definition + examples

- Make definition of symmetric difference more precise
- Add in set examples in description to facilitate learning of the
  concept
- Put "symmetric difference" in `<dfn>` tags

LenaBarinova committed:
Timestamps are converted to start of the days and filtered to b unique before calculating streaks

SaintPeter committed:
Merge pull request #6843 from erictleung/fix/clarify-symmetric-difference

Clarify symmetric difference definition + examples

raisedadead committed:
Fix text of the challenge with better wording

This commit changes the into text of the challenge with better wording,
as discussed in the issue thread.

Tested locally.

SaintPeter committed:
Merge pull request #6851 from raisedadead/fix/create-a-custom-heading

Fix text of the challenge with better wording

SaintPeter committed:
Add Wiki Sidebar

Commit:
Add challengeMap migrations

SaintPeter committed:
Merge pull request #6850 from LenaBarinova/fix/streak-broken-though-challenges-completed-everyday

Fix steak calculation to not break when challenges are completed everyday

Commit:
challenge/user router now works with challengeMap

alistermada committed:
Fix test for Iterate Through An Array With For Loop

Commit:
update copy and resolve merge conflict

Commit:
Merge branch 'Apercu-fix/imperative-wording' into staging

Commit:
Merge branch 'staging' of https://github.com/freecodecamp/freecodecamp into staging

LenaBarinova committed:
Green checkmarks added to Video and Zipline challenges view for completed challenges

SaintPeter committed:
Merge pull request #6856 from alistermada/fix/test-iterate-through-array

Fix test for Iterate Through An Array With For Loop

jorgearuv committed:
Challenges traslated to Spanish

jorgearuv committed:
Challenges traslated to Spanish

Understanding Uninitialized Variables, Understanding Case Sensitivity in Variables, Increment a Number with Javascript Updated

Previous correction completed. Challenges of Basic Javascript translated.

jorgeruvalcaba commented:
#6496 

Please check Spanish translations for challenges:

- [x] Escaping Literal Quotes in Strings.
- [x] Quoting Strings with Single Quotes.
- [x] Escape Sequences in Strings.
- [x] Concatenating Strings with Plus Operator.
- [x] Concatenating Strings with the Plus Equals Operator.
- [x] Constructing Strings with Variables.
- [x] Appending Variables to Strings.
- [x] Understand String Immutability
- [x] Word Blanks
- [x] Access Multi-Dimensional Arrays With Indexes

arialaurel committed:
Merge pull request #6351 from FreeCodeCamp/greenkeeper-dotenv-2.0.0

Update dotenv to version 2.0.0 🚀

Commit:
adding wiki as detached and integrating into .less folder

QuincyLarson committed:
Merge pull request #6858 from LenaBarinova/fix/green_checkmarks_missing_in_project_view

Green checkmarks added to Video and Zipline challenges view

SaintPeter committed:
Merge pull request #6861 from hallaathrad/feature/wiki-sidebar

adding wiki as detached and integrating into .less folder

QuincyLarson committed:
Merge pull request #6822 from pasosdeJesus/fix/ES-update-basic-bonfires

Verify and update translation to spanish of basic-bonfires

QuincyLarson committed:
Merge pull request #6803 from MatheusGodinho/add/PT-translation-json0apis-and-ajax

Challenges of Json APIS and Ajax translated.

Commit:
Commit uses challengeMap

SaintPeter committed:
Add language detection

greenkeeperio-bot committed:
chore(package): update gulp-rev to version 7.0.0

http://greenkeeper.io/

bugron committed:
Merge pull request #6809 from rfprod/add/RU-translation-react-projects

Russian Translation of React Projects.

bugron commented:
@jorgeruvalcaba please, [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) into one. And let us know when this PR is ready to be merged, thanks.

ankitaggarwal011 committed:
Update instructions and tests of change font-size challenge

arialaurel committed:
Merge pull request #6098 from FreeCodeCamp/greenkeeper-loopback-component-passport-2.0.0

Update loopback-component-passport to version 2.0.0 🚀

arialaurel committed:
Update react-motion spring argument signature

arialaurel committed:
Merge pull request #6620 from FreeCodeCamp/greenkeeper-react-motion-0.4.2

Update react-motion to version 0.4.2 🚀

arialaurel committed:
Merge pull request #6637 from FreeCodeCamp/greenkeeper-cheerio-0.20.0

Update cheerio to version 0.20.0 🚀

Commit:
Fix typos

Commit:
Fix isChallengeCompleted not returning a boolean

SaintPeter committed:
Fix progressTimestamps Update

Commit:
Merge pull request #6870 from SaintPeter/feature/reduce-query-load

Fix progressTimestamps Update

Commit:
Update completed challenge names

Commit:
fix for Flash message not displaying in detached map

Commit:
Filter out bad id's

SaintPeter committed:
Migrate invalid Challenge IDs

Commit:
Merge pull request #6874 from SaintPeter/feature/reduce-query-load

Migrate invalid Challenge IDs

Commit:
Fix lint errors

Commit:
Move idMap to utils so it can be used elsewhere

Commit:
Convert model instance to POJO earilier in migration change

Commit:
Make sure projects are always available

SaintPeter committed:
Merge pull request #6873 from hallaathrad/fix/flash-message-positioning

Fix for flash message not displaying in detached map

Commit:
Add validation to challenge completion
Change ajax requests to send and accept JSON
to preserve data types.
Fix typos

Commit:
update gear-up-for-success challenges with new social media outlets, no camper news

Commit:
make the "get set for" challenges optional

darthmeme committed:
fixed small typo in challenge title

raisedadead committed:
Fix the instructions in the Filter Array Challenge

This commit fixes the instruction text as discussed in the
issue thread.

Tested locally.

raisedadead committed:
Fix test for Create a bootstrap row challenge.

This commit fixes the tests to check if the `<div>` with `row` class is
actually nested below the `<h3>` tag.

Tested locally.

QuincyLarson committed:
Merge pull request #6854 from FreeCodeCamp/feature/reduce-query-load

Reduce query load/ add completed challenge validation

arialaurel committed:
Merge pull request #6883 from AkiraLaine/typo/testing-debugging

fixed small typo in challenge title

arialaurel committed:
Merge pull request #6888 from raisedadead/fix/filter-arrays-with-filter

Fix the instructions in the Filter Array Challenge

raisedadead committed:
Remove leftover question from JavaScript Lingo challenge

This commit :
1. Removes the leftover question from the JavaScript Lingo challenge
2. Fixes a warning in the Travis CI build.

Tested locally.

jamigibbs committed:
Fixes sidebar scrolling behavior on iOS, fixes #6227

jamigibbs committed:
Updating name to accurately describe parameter data type

Commit:
Fix cert look up causing crash

Commit:
Resequence front end projects and add Tribute Page project

Commit:
Update navber

Commit:
add faq and update about page

Commit:
update font awesome and move about page to all font awesome icons

erictleung committed:
Clarify instructions on saving object properties

QuincyLarson committed:
Merge pull request #6879 from FreeCodeCamp/feature/wiki-sidebar

Add Wiki Sidebar, Use Gatsby Generated Wiki

QuincyLarson committed:
Merge pull request #6915 from FreeCodeCamp/fix/cert-user-lookup

Fix cert look up causing crash

Commit:
Merge pull request #6922 from QuincyLarson/feature/new-social-challenges

Add new challenges and resequence front end projects

QuincyLarson committed:
Merge pull request #6864 from ankitaggarwal011/fix/update-instructions-tests-of-challenge

Update instructions and tests of change font-size challenge

Commit:
make email address a mailto link

Commit:
update sequence of about page content

Commit:
Merge pull request #6921 from FreeCodeCamp/feature/faq

Add FAQ to About page and make About page look much better

darthmeme committed:
Merge pull request #6911 from jamigibbs/fix/friendly-date-parameter

Updating name to accurately describe parameter data type

erictleung committed:
Add more formatting & emphasize palindrome details

- Put `true` and `false` in code tags
- Put `palindrome` in definition tags
- Add "Note" to further emphasize you'll need to remove all non-alphanumeric
  characters
- Put various forms of "race car" in code tags

ankitaggarwal011 committed:
Improve test cases of Record Collection challenge

Pwter committed:
Update jslingo.json

Commit:
Merge pull request #6908 from jamigibbs/fix/sidebar-scroll-ios

Fixes sidebar scrolling behaviour on iOS.

arialaurel committed:
remove fbjs dep

Commit:
Fix for bouncing sidecar's gap

Commit:
Merge pull request #6954 from ltegman/fix/remove-fbjs

remove fbjs dep

SaintPeter committed:
Fix issue with duplicate submits

Commit:
Merge pull request #6956 from SaintPeter/fix/duplicate-submits

Fix issue with duplicate submits

Commit:
Merge pull request #6945 from Pwter/patch-1

Update jslingo.json

SaintPeter committed:
Merge pull request #6953 from hallaathrad/fix/typos-and-style

Fix for bouncing sidecar's gap

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -188,7 +208,13 @@
+      "challengeType": 1,
+      "nameEs": "Entendiendo Variables Sin Inicializar",
+      "descriptionEs": [
+        "Cuando las variables de JavaScript son declaradas, ellas tienen un valor inicial de <code>undefined</code>. Si tu haces una operación matematica sobre una variable <code>undefined</code> tu resultado sera <code>NaN</code> el cual significa <dfn>\"Not a Number\"</dfn>. Si tu concatenas una cadena con una variable <code>undefined</code>, vas a obtener una <dfn>cadena</dfn> literal de <code ...
```
añadir después de Not a Numer -> es decir "No es un número"

jorgeruvalcaba replied:
¿Te parece bien que lo deje así y cuando acepten el cambio a inglés se cambie este challlenge? Lo digo para que no cambie mucho de ingles a español.

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -188,7 +208,13 @@
+      "descriptionEs": [
+        "Cuando las variables de JavaScript son declaradas, ellas tienen un valor inicial de <code>undefined</code>. Si tu haces una operación matematica sobre una variable <code>undefined</code> tu resultado sera <code>NaN</code> el cual significa <dfn>\"Not a Number\"</dfn>. Si tu concatenas una cadena con una variable <code>undefined</code>, vas a obtener una <dfn>cadena</dfn> literal de <code ...
+        "<h4>Instrucciones</h4>",
+        "Inicializa las tres variables <code>a</code>, <code>b</code>, y <code>c</code> con <code>5</code>, <code>10</code>, y <code>\"I am a\"</code> respectivamente de manera que no sea <code>undefined</code>."
```
Camibar "que no sea" por "que no sean"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -227,7 +253,18 @@
+        "En JavaScript todas las variables y nombres de funciones distinguen entre mayúsculas y minúsculas. Esto significa que la capitalización importa.",
+        "<code>MIVAR</code> no es lo mismo que <code>MiVar</code> o <code>mivar</code>. Esto hace posible tener múltiples variables distintas con el mismo nombre pero de diferente manera. Es fuertemente recomendado por el bien de la claridad, que tu <em>no</em> uses esta característica del lenguaje.",
+        "<h4>Mejor Práctica</h4>",
+        "Escribe nombres de variables en Javascript en <dfn>camelCase</dfn>. En <dfn>camelCase</dfn>, los nombres de variables con múltiples palabras tienen la primer palabra en minúsculas y la primera letra de cada palabra subsecuente es capitalizada.",
```
Cambiar "camelCase" por  "capitalizaciónCamello (camelCase)"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -410,7 +447,17 @@
+        "<code>i++</code>",
+        "es el equivalente de",
+        "<code>i = i + 1;</code>",
+        "<strong>Nota</strong><br>Toda la línea se convierte a <code>i++</code>, sin necesidad del signo igual.",
```
Me parece mejor cambiar "se convierte a" como "equivale a"

jorgearuv committed:
Understanding Uninitialized Variables, Understanding Case Sensitivity in Variables, Increment a Number with Javascript Updated

SaintPeter committed:
Merge pull request #6933 from ankitaggarwal011/fix/improve-test-cases-challenge

Improve test cases of Record Collection challenge

venky18 committed:
Updated the regex

updated the rgex  because "Target the Parent of an Element Using jQuery"  challenge was not accepting valid jquery
closes #6955

jamigibbs committed:
Fixes blank white wiki sidebar and scrolling

QuincyLarson committed:
Merge pull request #6978 from jamigibbs/fix/wiki-sidebar-ios

Fixes blank white wiki sidebar and scrolling in iOS

Commit:
Remove references to APIs from basic front end projects. Closes #6971

Commit:
Move academic honesty policy from wiki to main site

Commit:
Update hard-coded rate of challenge completion to reflect recent acceleration

Commit:
Add link to code of conduct and nonprofit project figure

jorgeruvalcaba commented:
Correcciones hechas @vtamara.

arialaurel committed:
coerce challengeType to string to fix video challenges

Commit:
Merge pull request #6980 from ltegman/fix/video-submissions

Fix video challenge submissions

Commit:
Lint pass

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -616,7 +684,17 @@
+        "Uno de tales operadores es el operador <code>+=</code>.",
+        "<code>miVar += 5;</code> añadirá <code>5</code> a <code>miVar</code>.",
+        "<h4>Instrucciones</h4>",
+        "Convierte las asignaciones para <code>a</code>, <code>b</code> y <code>c</code> para usar el operador <code>+=</code>."
```
"para usar" -> "usando"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -698,7 +785,16 @@
+        "multiplicará <code>miVar</code> por <code>5</code>. Esto puede ser reescrito como: ",
+        "<code>miVar *= 5;</code>",
+        "<h4>Instrucciones</h4>",
+        "Convierte las asignaciones para <code>a</code>, <code>b</code> y <code>c</code> para usar el operador <code>*=</code>."
```
"para usar" -> "usando2

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -777,7 +882,13 @@
+      "nameEs": "Convierte Celsius a Fahrenheit",
+      "descriptionEs": [
+        "Para probar tu aprendizaje, crearás una solucion \"desde cero\". Coloca tu código entre las líneas indicadas y este será probado contra multiples casos de prueba.",
+        "El algoritmo para convertir de Celsius a Fahrenheit es la temperatura en grados Celsius 9/5 veces, más 32.",
```
"es la temperatura en grados Celsius 9/5 veces, más 32" -> "consisten en multiplicar la temperatura en grados Celsius por 9/5 y al resultado agregarle 32"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -853,7 +964,18 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Escapando Citas Literales en Cadenas",
```
quote también se traduce como entrecomillar ( http://translation.babylon.com/spanish/to-english/entrecomillar/ ).  En mi humilde opinión a un lector le va a quedar más facil esta lección si traduces quote como "comillas".  Con esto el título sería "Escapar comillas en cadenas de texto"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -853,7 +964,18 @@
+      "nameEs": "Escapando Citas Literales en Cadenas",
+      "descriptionEs": [
+        "Cuando estas definiendo una cadena debes iniciar y terminar con comillas simples o dobles. ¿Qué pasa cuando necesitas una cita literal: <code>\"</code> o <code>'</code> dentro de tu cadena?",
+        "En JavaScript, tu puedes <dfn>escapar</dfn> una cita de considerarlo como un fin de la cita de cadena colocando una <dfn>barra invertida</dfn> (<code>\\</code>) antes de la cita.",
```
"una cita de considerarlo como un fin de la cita de cadena" -> "unas comillas para diferenciarlas de las comillas del final de la cadena"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -853,7 +964,18 @@
+        "Cuando estas definiendo una cadena debes iniciar y terminar con comillas simples o dobles. ¿Qué pasa cuando necesitas una cita literal: <code>\"</code> o <code>'</code> dentro de tu cadena?",
+        "En JavaScript, tu puedes <dfn>escapar</dfn> una cita de considerarlo como un fin de la cita de cadena colocando una <dfn>barra invertida</dfn> (<code>\\</code>) antes de la cita.",
+        "<code>var cadenaEjem = \"Alan dijo, \\\"Pedro esta aprendiendo JavaScript\\\".\";</code>",
+        "Esto le indiga a JavaScript que la cita siguiente no es el fin de la cadena, sino que debe aparecer dentro de la cadena. Así que si fueras a imprimir esto en la consola, tu obtendrías:",
```
indiga -> indica
que la cita -> que la comilla

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -884,7 +1006,16 @@
+      "challengeType": 1,
+      "nameEs": "Citando Cadenas con Comillas Simples",
+      "descriptionEs": [
+        "Los valores de <dfn>Cadenas</dfn> en JavaScript pueden ser escritos con comillas simples o dobles, siempre y cuando inicie y termine con el mismo tipo de cita. A diferencia de algunos lenguajes, comillas simples y dobles son funcionalmente identicas en Javascript.",
```
inicie y termine -> inicien y terminen

QuincyLarson committed:
Merge pull request #2 from hallaathrad/fix/update-about

Lint pass

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -853,7 +964,18 @@
+      "challengeType": 1,
+      "nameEs": "Escapando Citas Literales en Cadenas",
+      "descriptionEs": [
+        "Cuando estas definiendo una cadena debes iniciar y terminar con comillas simples o dobles. ¿Qué pasa cuando necesitas una cita literal: <code>\"</code> o <code>'</code> dentro de tu cadena?",
```
En programación comilla es ", mientras que apostrofe es '.  Sugiero cambiar "comillas simples" por "comillas" y "comillas simples" por "apostrofe".

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -884,7 +1006,16 @@
+      "descriptionEs": [
+        "Los valores de <dfn>Cadenas</dfn> en JavaScript pueden ser escritos con comillas simples o dobles, siempre y cuando inicie y termine con el mismo tipo de cita. A diferencia de algunos lenguajes, comillas simples y dobles son funcionalmente identicas en Javascript.",
+        "<code>\"Esta cadena tiene \\\"comillas dobles\\\" en esta\"</code>",
+        "El valor de usar una o la otra tiene que ver con la necesidad de <code>escapar</code> citas del mismo tipo. Si tu tienes una cadena con algunas citas dobles, esta puede ser difícil de leer y escribir. En su lugar, usar comillas simples:",
```
El valor -> La ventaja.
citas del mismo tipo -> comillas.
algunas citas dobles -> algunas comillas.
usar comillas simples -> usa apostrofes

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -884,7 +1006,16 @@
+        "Los valores de <dfn>Cadenas</dfn> en JavaScript pueden ser escritos con comillas simples o dobles, siempre y cuando inicie y termine con el mismo tipo de cita. A diferencia de algunos lenguajes, comillas simples y dobles son funcionalmente identicas en Javascript.",
+        "<code>\"Esta cadena tiene \\\"comillas dobles\\\" en esta\"</code>",
+        "El valor de usar una o la otra tiene que ver con la necesidad de <code>escapar</code> citas del mismo tipo. Si tu tienes una cadena con algunas citas dobles, esta puede ser difícil de leer y escribir. En su lugar, usar comillas simples:",
+        "<code>'Esta cadena tiene \"comillas dobles\" en esta. Y \"probablemente\" muchas de ellas.'</code>",
```
comillas dobles -> comillas

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -884,7 +1006,16 @@
+        "El valor de usar una o la otra tiene que ver con la necesidad de <code>escapar</code> citas del mismo tipo. Si tu tienes una cadena con algunas citas dobles, esta puede ser difícil de leer y escribir. En su lugar, usar comillas simples:",
+        "<code>'Esta cadena tiene \"comillas dobles\" en esta. Y \"probablemente\" muchas de ellas.'</code>",
+        "<h4>Instrucciones</h4>",
+        "Cambia la cadena dada de comillas dobles a simples y quita el escape."
```
comillas dobles -> comillas
simples -> apostrofes

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -915,7 +1046,15 @@
+      "challengeType": 1,
+      "nameEs": "Secuencias de Escape en Cadenas",
+      "descriptionEs": [
+        "Las comillas no son el único caracter que puede ser <dfn>escapado</dfn> dentro de una cadena. Aquí ahí una tabla de comunes secuencias de escape:",
```
comunes secuencias de escape -> secuencias de escape comunes

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -915,7 +1046,15 @@
+      "nameEs": "Secuencias de Escape en Cadenas",
+      "descriptionEs": [
+        "Las comillas no son el único caracter que puede ser <dfn>escapado</dfn> dentro de una cadena. Aquí ahí una tabla de comunes secuencias de escape:",
+        "<table class=\"table table-striped\"><thead><tr><th>Código</th><th>Salida</th></tr></thead><tbody><tr><td>\\'</td><td>comilla simple</td></tr><tr><td>\\\"</td><td>comilla doble</td></tr><tr><td>\\\\</td><td>barra invertida</td></tr><tr><td>\\n</td><td>nueva línea</td></tr><tr><td>\\r</td><td>retorno de carro</td></tr><tr><td>\\t</td><td>tabulación</td></tr><tr><td>\\b</td><td>retroceso</ ...
```
comilla simple -> apostrofe
comilla doble -> comilla

Commit:
Merge pull request #6979 from QuincyLarson/fix/update-about

Add academic honesty page and improve about view

arialaurel committed:
Merge pull request #6976 from venky18/patch-3

Updated the regex

justinian336 committed:
Adapt translation according to the new curriculum

arialaurel committed:
Merge pull request #6811 from pasosdeJesus/fix/ES-update-advanced-bonfires

Verified and updated translation to spanish of advanced-bonfires.

arialaurel committed:
Merge pull request #6890 from raisedadead/fix/video-challenges-jslingo

Remove leftover question from JavaScript Lingo challenge

justinian336 committed:
Updating the translation to the new curriculum.

justinian336 committed:
Updating the translation according to the new curriculum

justinian336 committed:
Update the translation according to the new curriculum

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -958,7 +1097,16 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Concatenando Cadenas con el Operador Más",
```
Solemos traducir gerundios al comienzo de título en infinitivo, sería "Concatenar" en lugar de "Concatendando"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -998,7 +1146,14 @@
+      "challengeType": 1,
+      "nameEs": "Concatenando Cadenas con el Operador Mas Igual",
+      "descriptionEs": [
+        "Nosotros también podemos usar el operador <code>+=</code> para <dfn>concatenar</dfn> una cadena en el extremo de una variable de cadena existente. Esto puede ser muy útil para romper una cadena larga sobre varias líneas.",
```
"en el extremo" -> "al final"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1091,7 +1252,13 @@
+      "challengeType": 1,
+      "nameEs": "Anexando Variables a Cadenas",
+      "descriptionEs": [
+        "Al igual que podemos construir una cadena sobre múltipes líneas fuera de las cadenas <dfn>literales</dfn>, nosotros podemos ademas anexar variables a una cadena usando el operador más igual (<code>+=</code>).",
```
"sobre múltiples líneas fuera de las cadenas literales" -> " en múltiples líneas a partir de cadenas literales"

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -998,7 +1146,14 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Concatenando Cadenas con el Operador Mas Igual",
```
Concatenando -> Concatenar

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1044,7 +1199,13 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Construyendo Cadenas con Variables",
```
Construyendo -> Construye

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1220,7 +1387,17 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Comprendiendo la Inmutabilidad de Cadenas",
```
Comprendiendo -> Comprender

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1387,7 +1564,13 @@
+      "challengeType": 1,
+      "nameEs": "Espacios en Blanco de Palabras",
+      "descriptionEs": [
+        "Ahora usaremos nuestro conocimiento de cadenas para construir una \"<a href='https://en.wikipedia.org/wiki/Mad_Libs' target='_blank'>Mad Libs</a>\" estilo juego de palabras que llamaremos \"Espacios en Blanco de Palabras\". Vas a crear unas oraciones estilo (opcionalmente graciosa) \"Llena los espacios en blanco\".",
```
una Mad Libs estilo juego de palabras -> un juego de palabras al estilo Mad Libs

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1387,7 +1564,13 @@
+      "nameEs": "Espacios en Blanco de Palabras",
+      "descriptionEs": [
+        "Ahora usaremos nuestro conocimiento de cadenas para construir una \"<a href='https://en.wikipedia.org/wiki/Mad_Libs' target='_blank'>Mad Libs</a>\" estilo juego de palabras que llamaremos \"Espacios en Blanco de Palabras\". Vas a crear unas oraciones estilo (opcionalmente graciosa) \"Llena los espacios en blanco\".",
+        "Además, necesitarás proporcionar cadenas adicionales, las cuales no cambiarán entre las palabras proporcionadas.",
```
Flata traducir una línea antes de esta: 
You will need to use string operators to build a new string, <code>result</code>, using the provided variables: <code>myNoun</code>, <code>myAdjective</code>, <code>myVerb</code>, and <code>myAdverb</code>.

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1578,7 +1761,15 @@
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Accediendo a Vectores Multi-Dimensionales con Indices",
```
Acceder

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1578,7 +1761,15 @@
+      "challengeType": 1,
+      "nameEs": "Accediendo a Vectores Multi-Dimensionales con Indices",
+      "descriptionEs": [
+        "Una manera de pensar un vector <dfn>multi-dimensional</dfn>, es como un <em>vector de vectores</em>. Cuando usas brackets para accesar a tu vector, el primer conjunto de brackets se refiere a las entradas en el más exterior vector y cada nivel subsecuente de brackets se refiere al siguiente nivel de entradas dentro.",
```
brackets -> corchetes
accesar -> acceder
más exterior vector -> vector más externo
siguiente nivel de entradas dentro -> siguiente nível de vectores internos

vtamara commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -1578,7 +1761,15 @@
+        "<strong>Ejemplo</strong>",
+        "<blockquote>var vec = [<br>    [1,2,3],<br>    [4,5,6],<br>    [7,8,9],<br>    [[10,11,12], 13, 14]<br>];<br>vec[0]; // es igual [1,2,3]<br>vec[1][2]; // es igual 6<br>vec[3][0][1]; // es igual 11</blockquote>",
+        "<h4>Instrucciones</h4>",
+        "Lee de <code>myArray</code> usando la notación bracket de modo que myData sea igual a <code>8</code>"
```
bracket -> corchete

bugron committed:
Merge pull request #6989 from justinian336/Audit/ES-Translation-of-nodejs-and-expressjs.json

Update the translation according to the new curriculum

bugron committed:
Merge pull request #6985 from justinian336/Audit/ES-Translation-of-mongodb.json

Updating the translation according to the new curriculum

bugron committed:
Merge pull request #6984 from justinian336/Audit/ES-Translation-of-git.json

Updating the translation to the new curriculum.

bugron committed:
Merge pull request #6983 from justinian336/Audit/ES-Translation-of-dynamic-web-applications.json

Adapt translation according to the new curriculum

justinian336 committed:
Audit ES Translation of data visualization certificate

justinian336 committed:
Audit ES Translation of back-end-development-certificate.json

vtamara committed:
Update translation to spanish of basic-ziplines. Thanks for corrections to @justinian336
Closes #6497

vtamara committed:
Update and verify spanish translation of advanced-ziplines. Thanks for corrections to @justinian336
Closes #7003

jorgeruvalcaba commented:
Correcciones hechas @vtamara, gracias!

erictleung committed:
Clarify join method challenge with better example

arialaurel committed:
Merge pull request #6643 from FreeCodeCamp/greenkeeper-nodemailer-2.1.0

Update nodemailer to version 2.1.0 🚀

arialaurel committed:
Merge pull request #6862 from FreeCodeCamp/greenkeeper-gulp-rev-7.0.0

Update gulp-rev to version 7.0.0 🚀

arialaurel committed:
Merge pull request #6925 from erictleung/fix/further-clarify-dot-operator

Clarify instructions on saving object properties

arialaurel committed:
Merge pull request #6928 from erictleung/fix/format-palindromes-challenge

Add more formatting & emphasize palindrome details

MDAWebDev committed:
fixed Moore's Law question

arialaurel committed:
Merge pull request #7011 from MDAWebDev/pre-staging

fixed Moore's Law question (new)

Commit:
Fix next/current challenge routes
prevent bad id's from throwing

jorgearuv committed:
Previous correction completed. Challenges of Basic Javascript translated.

jorgeruvalcaba commented:
Challenges translated to Spanish:

- [x] Shopping List.
- [x] Passing Values to Functions with Arguments.
- [x] Global Scope and Functions.
- [x] Local Scope and Functions.
- [x] Global vs. Local Scope in Functions.
- [x] Return a Value from a Function with Return.
- [x] Assignment with a Returned Value.
- [x] Stand in Line.
- [x] Comparison with the Equality Operator.
- [x] Comparison with the Strict Equality Operator.
- [x] Comparison with the Inequality Operator.

vtamara committed:
Update and verify translation of intermeditate-bonfires
Closes #6503

QuincyLarson committed:
Merge pull request #7014 from pasosdeJesus/fix/ES-update-intermediate-ziplines

Update and verify translation to spanish of intermeditate-ziplines

QuincyLarson committed:
Merge pull request #7013 from FreeCodeCamp/fix/challenge-routes

Fix next/current challenge routes

Commit:
Remove console log

QuincyLarson committed:
Merge pull request #7021 from FreeCodeCamp/fix/remove-console-log

Remove console log

QuincyLarson committed:
Merge pull request #6995 from justinian336/Audit/ES-Translation-of-back-end-development-certificate.json

Audit ES Translation of back-end-development-certificate.json

QuincyLarson committed:
Merge pull request #6993 from justinian336/Audit/ES-Translation-of-data-visualization-certificate.json

Audit ES Translation of data visualization certificate

bugron committed:
Merge pull request #7008 from erictleung/fix/clarify-join-method

Clarify join method challenge with better example

prayagverma committed:
Fix typo in api-projects

darthmeme committed:
Merge pull request #7024 from pra85/fix/typo-backend-api-projects

Fix typo in api-projects

ankitaggarwal011 committed:
Improve grammar of instructions for font-awesome bootstrap challenge

darthmeme committed:
Merge pull request #7025 from ankitaggarwal011/fix/improve-grammar-instructions-bootstrap-challenge

Improve grammar of instructions for font-awesome bootstrap challenge

justinian336 committed:
Audit ES Translation of front-end-development-certificate.json

bugron committed:
Merge pull request #7028 from justinian336/Audit/ES-Translation-of-front-end-development-certificate.json

Audit ES Translation of front-end-development-certificate.json

bugron commented:
@jorgeruvalcaba again, please, [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) into one.

arialaurel committed:
Merge pull request #6891 from raisedadead/fix/create-a-bootstrap-row

Fix test for Create a bootstrap row challenge.

Commit:
add advanced project requirement to claim front end certificate challenge

jorgeruvalcaba commented:
@bugron Done.

Commit:
Merge pull request #7047 from QuincyLarson/fix/front-end-cert-description

add advanced project requirement to claim front end certificate challenge

erictleung committed:
Clarify Counting Cards instructions & add tests

- Clarify instructions by changing table column "Value" to "Count Change"
- Add two more tests with mix of 7, 8, 9 and other card values

erichartline committed:
Fixed typos on two video challenge pages

bugron commented:
@jorgeruvalcaba unfortunately, it's not, still you have four commits. See these articles: [one](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/), [two](https://davidwalsh.name/squash-commits-git). If you still need help go to our https://gitter.im/FreeCodeCamp/HelpContributors Gitter chat room, thanks.

arialaurel committed:
Merge pull request #7055 from wildlifehexagon/branch/fix-video-page-typos

Fixed typos on two video challenge pages

raisedadead committed:
Update wiki link for twitch JSON Challenge

bugron committed:
Merge pull request #7061 from raisedadead/fix/twitchtv-JSON-api

Update wiki link for twitch JSON Challenge

Commit:
fix-odd-spacing

Addresses the JS challenge generator trying to wrap the `<ol>` element inside of a paragraph (and obviously failing)

Closes #6840

bugron committed:
Merge pull request #7004 from pasosdeJesus/fix/ES-update-basic-ziplines

Update translation to spanish of basic-ziplines

bugron committed:
Merge pull request #7005 from pasosdeJesus/fix/ES-update-advanced-ziplines

Update and verify spanish translation of advanced-ziplines

SaintPeter committed:
Merge pull request #7051 from erictleung/fix/clarify-ambiguity-for-counting-cards

Clarify Counting Cards instructions & add tests

SaintPeter committed:
Merge pull request #7066 from hallaathrad/fix/odd-spacing-in-challenge

Fix odd spacing in JS challenge

jorgearuv committed:
Challenges of Basic Javascript completely translated. Closes #6496

jorgearuv committed:
Merge branch 'add/ES-translation-basic-javascript' of https://github.com/jorgeruvalcaba/FreeCodeCamp into add/ES-translation-basic-javascript

jorgeruvalcaba commented:
Translation of all challenges completed.

vtamara commented:
@jorgeruvalcaba que tal si: 
1 copias el archivo que tradujiste con las correcciones que llevas a un directorio (digamos /tmp).  
2. Actualizas tu bifurcacion de Fcc con
git pull --rebase upsrtream staging (asegurando antes que agregaste el remoto upstream apuntando al fcc oficial, mira rebasing en archivo contributing.md).  Si esto suena dificil borra en github tu bifurcacion de fcc y vuelvela a hacer.

vtamara commented:
3. Clona tu bifurcacion actualizada
4. Inicia una rama
5. En la nueva rama copia el archivo basic-javascript.json que guardaste en paso 1 a donde corresponde
6. Haz un commit y un push

vtamara commented:
7. Haz un nuevo pull request que seguire re isando donde iba
8. Haces correcciones finales y te explicare como integrarlas.

jorgeruvalcaba commented:
Listo @vtamara. El nuevo pull request es #7091
